gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #### finding multivariate pdf and saving the LATEXIFY figures ####
</pre><p>First split this cell into multiple cells, keep similar code in one cell. for example that two plotting function can be in one cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>